src/hotspot/share/runtime/globals.hpp

```diff
@@ -2050,6 +2050,9 @@ const int ObjectAlignmentInBytes = 8;
           "Minimal number of elements in a sorted collection to prefer"     \
           "binary search over simple linear search." )                      \
                                                                             \
+  product(bool, UseAltSubstitutabilityMethod, false,                        \
```

Comment:
We might need this flag on/off for core-libs tests in addition to runtime tests.